doc/shortcut.md

```diff
@@ -0,0 +1,50 @@
+# Shortcut
+
+### What is it?
+Shortcut is a component that represents a shortcut to some action
```

Comment:
Sugestão: sempre usar ponto final (ou dois pontos, quando for o caso) na documentação (exceto nos títulos). Quando não usamos, pode parecer para quem está lendo que a frase continua na próxima linha, ou mesmo que falta algo.

Além disso, se alguém usa leitor de tela para ler a documentação, sem uma pontuação, o leitor pode "pular" para o título ou o outro parágrafo sem pausa, o que pode dificultar o entendimento.

Reply:
Boa! Vou ajustar isso :)